### ISSUE TITLE
Ketdelle's Hit Points are wrong

### ISSUE THREAD

**Issue by trey-kirk-sp** [mr-ice/maptool-macros#38]
They add +1 per dwarfiness. Needs to read this modifier when building HP
data.modifiers.race

```
 {
                    "availableToMulticlass": true,
                    "bonusTypes": [],
                    "dice": null,
                    "duration": null,
                    "entityId": null,
                    "entityTypeId": null,
                    "fixedValue": 1,
                    "friendlySubtypeName": "Hit Points per Level",
                    "friendlyTypeName": "Bonus",
                    "id": "1873",
                    "isGranted": true,
                    "modifierSubTypeId": 752,
                    "modifierTypeId": 1,
                    "requiresAttunement": false,
                    "restriction": "",
                    "statId": null,
                    "subType": "hit-points-per-level",
                    "type": "bonus",
                    "value": 1,
                    "componentId": 122,
                    "componentTypeId": 1960452172
                }
```

**Comment by havoclad** [comment]
Hill Dwarf's only

Dwarven ToughnessBR, pg. 20
Your hit point maximum increases by 1, and it increases by 1 every time you gain a level.

**Comment by trey-kirk-sp** [comment]
Fixed in hash a4fe2431
